yasa/plotting.py

```diff
@@ -54,79 +41,96 @@ def plot_hypnogram(hypno, sf_hypno=1 / 30, lw=1.5, fill_color=None, ax=None):
     --------
     .. plot::
 
-        >>> import yasa
-        >>> import numpy as np
+        >>> from yasa import Hypnogram
         >>> import matplotlib.pyplot as plt
-        >>> hypno = np.loadtxt("https://github.com/raphaelvallat/yasa/raw/master/notebooks/data_full_6hrs_100Hz_hypno_30s.txt")
-        >>> fig, ax = plt.subplots(1, 1, figsize=(7, 3), constrained_layout=True)
-        >>> ax = yasa.plot_hypnogram(hypno, fill_color="gainsboro", ax=ax)
+        >>> values = 4 * ["W", "N1", "N2", "N3", "REM"] + ["ART", "N2", "REM", "W", "UNS"]
+        >>> hyp = Hypnogram(values, freq="24min").upsample("30s")
```

Comment:
That said, the first plot looks very nice so feel free to just discard this comment

Reply:
I feel the same. I think the `freq` -> `upsample` approach to get a long Hypnogram might be confusing, but the benefits outweigh the downsides. I like it because it gives a super-stereotyped hypnogram that is easy to read and also might be useful to make for a talk or something. It also promotes the `upsample` method a bit and shows its usefulness, even if in a weird way!

Reply:
On second thought, should we add one more example that is more straightforward? Just `simualte_hypno` and `plot_hypnogram` with no fluff?

Reply:
Sounds good for keeping this one in and adding a simpler example first 👍